tox.ini

```diff
@@ -13,7 +13,7 @@ basepython = python3
 basepython = python3
 deps=
     sphinx
-whitelist_externals = make
+allowlist_externals = make
```

Comment:
Ditto. Not a bad change, but just double-checking that it doesn't really have anything to do with the change in this PR.

Reply:
This is the commit: https://github.com/python/pyperformance/pull/351/commits/f47ab7c5d0392cb8c71b53a9dc557076e1ab7bf7

Reply:
Dropped, I'll raise a new PR